.github/workflows/pullrequest.yml.template

```diff
@@ -0,0 +1,76 @@
+# Name of the PlaceHolder                 -  Description
+########################################################################################################################
+# <Name of the WorkFlow>                  - Workflow name. example: Bikes
+# <Path to the service>                   - Path to the service where the developer would like to trigger the Flow/ action on pull request - Example: samples/BikeSharingApp/Bikes. 

```

Comment:
This is really the path to the service that must be modified in order for the workflow to trigger.

Reply:
Updated.